pkg/webui/console/components/webhook-form/index.js

```diff
@@ -151,6 +153,7 @@ export default class WebhookForm extends Component {
           component={Input}
           required
         />
+        <Form.Field name="downlink_api_key" title={m.downlinkAPIKey} component={Input} code />
```

Comment:
Just for my understanding: it is not a required value then?

Reply:
Afaik we need the same validation here, probably both API and front-end, as we have for the application link API key.

Reply:
The field is not required (as this feature is more like syntactic sugar for the webhook operations). On the API side, I've left it without validation (as generally we do only minimum length >= 1, no upper limit), and on the Console side it will be validated like the other API keys when present.